.github/workflows/test_stable.yml

```diff
@@ -0,0 +1,70 @@
+name: Test stable
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  Test:
+    defaults:
+      run:
+        shell: bash -l {0}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ["windows-latest"] #, "ubuntu-latest", "macos-latest"]
+        python-version: ["3.10"] # fix tests to support older versions
+        include:
+          - os: windows-latest
+            label: win-64
+            prefix: c:/Miniconda3/envs/hydromt_wflow
+
+    name: ${{ matrix.label }} - py${{ matrix.python-version }}
+    runs-on: ${{ matrix.os }}
+    concurrency:
+      group: ${{ github.workflow }}-${{ matrix.python-version }}-${{ github.ref }}
+      cancel-in-progress: true
+
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v3
+
+      - name: Setup Mambaforge
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+          miniforge-variant: Mambaforge
+          miniforge-version: latest
+          activate-environment: hydromt_wflow
+          use-mamba: true
+
+      - name: Generate env spec
+        run: pip install tomli && python make_env.py test --py-version ${{ matrix.python-version }}
+
+      - name: Set cache date
+        run: echo "DATE=$(date +'%Y%m%d')" >> $GITHUB_ENV
+
+      - uses: actions/cache@v3
+        with:
+          path: ${{ matrix.prefix }}
+          key: ${{ matrix.label }}-conda-${{ hashFiles('environment.yml') }}-${{ env.DATE }}
+        id: cache
+
+      - name: Update environment
+        run: mamba env update -f environment.yml
+        if: steps.cache.outputs.cache-hit != 'true'
+
+      - name: Install HydroMT-Wflow plugin
+        run: pip install .
+
+      - name: Run tests
+        run: python -m pytest --verbose --cov=hydromt_wflow --cov-report xml
+
+      - name: Check style
+        run: black --check .
```

Comment:
this should not be necessary here as that is tsted by the pre-commit/lining action

Reply:
Same.